main_logic/core.py

```
@@ -1262,452 +1262,452 @@ async def start_session(self, websocket: WebSocket, new=False, input_mode='audio
 
         # 标记正在启动（使用计数器，避免并发 start_session 的 finally 互相覆盖）
         self._starting_session_count += 1
+        try:
         
-        # 回收残留的热切换资源，防止 main + pending + new-main 叠到 >2 个 session
-        await self._cleanup_pending_session_resources()
-        await self._reset_preparation_state(clear_main_cache=False)
+            # 回收残留的热切换资源，防止 main + pending + new-main 叠到 >2 个 session
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)